app/src/ui/dropdown-select-button.tsx

```diff
@@ -20,16 +23,19 @@ interface IDropdownSelectButtonProps<T extends string> {
   readonly options: ReadonlyArray<IDropdownSelectButtonOption<T>>
 
   /** The selection option value */
-  readonly selectedValue?: T
+  readonly checkedOption?: T
```

Comment:
Since the menu items uses the term "selected" to denote the "Highlighted" or "Focused" value, I changed this to "checked" to make that distinction clearer.